### PR TITLE
add support to add example on field level

### PR DESCRIPTION
**Pull request by dimzst**
value must be a json value (refer to extract_example.txt for more
example)
usage example:
```
// OwnerName is the name of the account's owner.
//
// +gunk openapiv2.Schema{
//         JSONSchema: openapiv2.JSONSchema{
//             Example: `"John Doe"`,
//         },
// }
OwnerName string `pb:"21" json:"owner_name"`
```

### CODE REVIEW

File: docgen/docgen.go [hunk 1]
```diff
@@ -20,6 +21,7 @@ import (
 const modeAppend = "append"
 
 func main() {
+	log.Println("SOMETHING")
```
Comment (karelbilek) [inline docgen/docgen.go:24]: remove this

File: docgen/parser/parser.go [hunk 2]
```diff
@@ -232,12 +232,18 @@ func parseMessages(
 	// convert each proto message to our Message representation
 	for i, m := range messages {
 
-		fixedExample := ""
-		schemaI := proto.GetExtension(m.GetOptions(), options.E_Openapiv2Schema)
-		if schemaI != nil {
-			schema := schemaI.(*options.Schema)
-			if schema != nil {
-				fixedExample = schema.Example
+		var fixedExample map[string]interface{}
```
Comment (karelbilek) [inline docgen/parser/parser.go:235]: What's the point of this change?

Comment (karelbilek) [reply docgen/parser/parser.go:235]: Ah I see below

Comment (dimzst) [reply docgen/parser/parser.go:235]: Yup, that is to support nested message

**Comment by karelbilek** [comment]
great job

**Pull request merged**